Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:latest AS build
```

Comment:
would be nice to use _some_ labels here that follow the [label-schema](http://label-schema.org/rc1/) . Using labels makes it friendlier to others that want to get more information about the container itself, when it was built, and from what commit.

Reply:
added relevant labels